app/controllers/course-admin/code-examples-index.ts

```
@@ -0,0 +1,61 @@
+    
+    return stagesWithAnalyses;
+  }
+  
+  @action
+  onLanguageChange(event: Event): void {
+    const select = event.target as HTMLSelectElement;
+    const languageSlug = select.value;
+    
+    this.router.transitionTo({
+      queryParams: {
+        language: languageSlug
```

ESLint

[error] Insert `,`

(prettier/prettier)